scripts/get-sources

```diff
@@ -105,6 +100,8 @@ fi
 
 if [ "$verify" == "true" ]; then
     echo "--> Verifying tags..."
+    KEYRING_DIR_GIT="$KEYRING_DIR_GIT/$REPO"
```

Comment:
Keyring for git tags should be separate from the one for tarballs etc.

Reply:
`$BUILDER_DIR/keyring/git/$REPO`?